searx/botdetection/limiter.toml

```diff
@@ -10,6 +10,24 @@ x_for = 1
 ipv4_prefix = 32
 ipv6_prefix = 48
 
+[botdetection.ip_lists]
+
+# In the limiter, the ip_lists method has priority over all other methods -> if
+# an IP is in the pass_ip list, it has unrestricted access and it is also not
+# checked if e.g. the "user agent" suggests a bot (e.g. curl).
+
+pass_ip = [
+  '51.15.252.168',         # IPv4 of check.searx.space
+  '2001:bc8:407:301::1c',  # IPv6 of check.searx.space
```

Comment:
@unifox: is the IPv4 and IPv6 of searx.space's crawler correct?

    $ nslookup check.searx.space
    ...
    Non-authoritative answer:
    Name:	check.searx.space
    Address: 51.15.252.168
    Name:	check.searx.space

Reply:
I'm not sure, for some reason the IPv6 address doesn't respond to the ping. @dalf told me that he will update the record with the correct IP addresses.

Reply:
@dalf @mrpaulblack AFIK you are the maintainers/owners of the searx.space and the services (right?) .. what IPs does the crawler use?

I need to know these IPs to add them to the passlist of this PR.

In the ["Issue: Add a new instance"](https://github.com/searxng/searx-instances/issues/new?assignees=&labels=instance+add%2Cinstance&projects=&template=add-instance.yaml&title=Add+https%3A%2F%2F%3Cinstance+URL%3E) template I read:

     Your instance will receive requests from the host check.searx.space
     (the IP addresses of this host may change in the future).

From where can I get the IPs? .. nslookup gives me `51.15.252.168` but I have never seen a request from this IP on my instance .. further, every few ours I get different IPv6 addresses .. here is a list of what I have seen in the last two days:

- `2001:bc8:407:301::1c`
- `2001:bc8:47a4:1a55::1`
- `2001:bc8:407:301::1c`

Is it a DNS issue or does the crawler of check.searx.space have a range of IPv4 & IPv6 addresses?